1. Mexiko Create filenames filename-mappings and infofiles.py

```diff
@@ -326,13 +326,15 @@ def create_infofiles(row, filenames_file, not_ok_file):
             personnamn_not_even = True
             infotext += "|depicted people     = " + row["Personnamn / avbildad"] + "\n"
         else:
+            if "Linné, Sigvald" in row["Personnamn / avbildad"]:
```

Comment:
Ah, OK. 
Fixed with a slight modification - 'in' instead of '=' in a68277ccd69763813c1561b96874e8eff9297f17